src/components/TopMenu.tsx

```diff
@@ -176,4 +180,30 @@ function RewardsButton({
   ) : null
 }
 
+interface SDLPriceProps {
+  sdlPrice: number | undefined
+}
+
+function SDLPrice({ sdlPrice }: SDLPriceProps): ReactElement | null {
+  if (sdlPrice !== undefined) {
+    const SUSHI_WETH_SDL_POOL_URL =
+      "https://app.sushi.com/analytics/pools/0x0c6f06b32e6ae0c110861b8607e67da594781961?chainId=1"
+    return (
+      <Button
+        variant="contained"
+        color="info"
+        data-testid="sdlPriceButton"
+        href={SUSHI_WETH_SDL_POOL_URL}
+        target="_blank"
+        startIcon={<SaddleLogo width={20} height={20} />}
+        style={{ minWidth: 250 }}
+      >
+        {`SDL Price: ${sdlPrice}`}
```

Comment:
Since this takes up a lot of space, maybe we can just format it as `$0.25` (dollar sign, no text, 2 decimal places) since we already have the icon there